src/vs/workbench/browser/parts/editor/editorGroupsControl.ts

```diff
@@ -2013,6 +2143,32 @@ export class EditorGroupsControl extends Themable implements IEditorGroupsContro
 			}
 		});
 
+		// Layout centered Editor
+		const doCentering =
+			this.layoutVertically &&
+			this.editorGroupService.getStacksModel().groups.length === 1 &&
+			this.partService.isLayoutCentered() &&
+			this.visibleEditors[Position.ONE] instanceof TextResourceEditor;
```

Comment:
Yeah definetely remove this instanceof check, since the feature does not behave like it should with it. @bpasero was unable to try it out.

Reply:
done.